charts/podiumd/values.yaml

```diff
@@ -700,7 +700,7 @@ opennotificaties:
   persistentVolume:
     volumeAttributeShareName: opennotificaties
   image:
-    tag: "1.8.4"
+    tag: "1.8.2"

```

Comment:
When you set both the PodiumD Helm and application versions in the `Chart.yaml` to `4.1.3` (next patch), then we can merge this PR and the PodiumD Helm patch will be built and is available for deployment 👍